routers/repo/issue.go

```diff
@@ -896,6 +913,11 @@ func ViewIssue(ctx *context.Context) {
 	ctx.Data["SignInLink"] = setting.AppSubURL + "/user/login?redirect_to=" + ctx.Data["Link"].(string)
 	ctx.Data["IsIssuePoster"] = ctx.IsSigned && issue.IsPoster(ctx.User.ID)
 	ctx.Data["IsIssueWriter"] = ctx.Repo.CanWriteIssuesOrPulls(issue.IsPull)
+	ctx.Data["IsRepoAdmin"] = ctx.IsSigned && ctx.Repo.IsAdmin() && ctx.User.IsAdmin
+	ctx.Data["IsWriter"] = ctx.IsSigned && ctx.Repo.CanWrite(models.UnitTypeIssues) && ctx.User.IsAdmin
+	ctx.Data["IsRepoAdmin"] = ctx.IsSigned && (ctx.Repo.IsAdmin() || ctx.User.IsAdmin)
+	ctx.Data["IsRepoWriter"] = ctx.IsSigned && (ctx.Repo.CanWrite(models.UnitTypeIssues) || ctx.User.IsAdmin)
```

Comment:
What is the difference between `IsWriter` and `IsRepoWriter`? Why are both needed?

Reply:
Turns out `IsWriter` was redundant and not used anywhere